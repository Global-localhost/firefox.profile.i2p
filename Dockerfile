FROM eyedeekay/tbb-docker:torbrowser
COPY firefox.profile.i2p/bookmarks.html \
    /home/anon/tor-browser_en-US/Browser/TorBrowser/Data/Browser/profile.default/bookmarks.html
COPY firefox.profile.i2p/user.js \
    /home/anon/tor-browser_en-US/Browser/TorBrowser/Data/Browser/profile.default/user.js
CMD /home/anon/tor-browser_en-US/Browser/start-tor-browser
