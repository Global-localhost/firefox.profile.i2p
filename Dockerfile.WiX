FROM ubuntu:16.04
RUN apt-get update && apt-get dist-upgrade -y
RUN apt-get install -y \
    ca-certificates \
    gcab \
    gir1.2-libmsi* \
    libmsi* \
    libgcab* \
    libqxp* \
    make \
    msitools \
    wget \
    python-dev \
    python-pip \
    python-gi \
    python-gobject \
    gir1.2-glib-2.0 \
    gir1.2-libmsi0 \
    gir1.2-libgcab-1.0
RUN wget https://github.com/sk1project/wixpy/releases/download/v0.1/python-wix.py-0.1_ubuntu_16.04_amd64.deb -O python-wix.py-0.1.deb
RUN apt-get install -y ./python-wix.py-0.1.deb
RUN pip install --upgrade pip
#RUN pip install --upgrade cabarchive wix.py cffi \
#    https://github.com/mindcollapse/libmsi-python/archive/master.zip \
#    https://github.com/sk1project/wixpy/archive/master.zip
RUN pip install --upgrade cabarchive wix.py cffi
RUN pip install --upgrade https://github.com/sk1project/wixpy/archive/master.zip
RUN adduser --home /home/profbuild --disabled-password --gecos 'profbuild,,,,' profbuild
COPY . /home/profbuild/working
RUN chown -R profbuild:profbuild /home/profbuild/working
USER profbuild
WORKDIR /home/profbuild/working
RUN make wix && wix.py --xml_only ./wixpy.json
CMD cat *.wxs

