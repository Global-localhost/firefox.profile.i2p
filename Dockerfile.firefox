FROM debian:sid
RUN apt-get update
RUN apt-get dist-upgrade -y
RUN apt-get install -y firefox-esr
COPY firefox.launchers/gnulinux /home/user/firefox.profile
WORKDIR /home/user/firefox.profile
CMD ./install.sh run
