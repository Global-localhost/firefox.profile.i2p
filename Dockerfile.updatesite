FROM eyedeekay/eephttpd
COPY . /opt/eephttpd/www
USER eephttpd
CMD eephttpd -f /usr/src/eephttpd/etc/eephttpd/eephttpd.conf \ -i -n i2pbrowser-updates -s /opt/eephttpd/ -sh=sam-host -sp=7656 -r
