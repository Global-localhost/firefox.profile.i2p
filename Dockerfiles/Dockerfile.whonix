FROM eyedeekay/whonix:unstable
ENV TOR_SKIP_LAUNCH=1
ENV TOR_SKIP_CONTROLPORTTEST=1
ENV TOR_HIDE_BROWSER_LOGO=1
ENV TOR_HIDE_UPDATE_CHECK_UI=1
ENV TOR_NO_DISPLAY_NETWORK_SETTINGS=1
USER root
RUN apt-get update && apt-get dist-upgrade -y
RUN apt-get install -y tb-starter tb-updater
USER user
WORKDIR /home/user
CMD update-i2pbrowser --devbuildpassthrough && i2pbrowser
