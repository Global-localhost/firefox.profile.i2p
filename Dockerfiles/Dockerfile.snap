FROM ubuntu:18.04
#FROM debian:sid
RUN apt-get update && apt-get dist-upgrade -y
RUN apt-get install snapcraft* -y
RUN adduser --home /home/snap --disabled-password --gecos 'snap,snap,snap,snap,snap' snap
#COPY firefox.launchers/snap /home/snap/snap
COPY . /home/snap/snap
WORKDIR /home/snap/snap
RUN chown snap:snap -R /home/snap/snap
#RUN sed -i 's|firefox.launcher/snap/gnulinux|gnulinux|g' /home/snap/snap/snapcraft.yaml
RUN snapcraft
CMD bash -i
